src/main/java/com/google/devtools/build/lib/sandbox/SandboxModule.java

```diff
@@ -272,9 +272,9 @@ private void setup(CommandEnvironment cmdEnv, ExecutorBuilder builder)
     }
 
     PathFragment windowsSandboxPath = PathFragment.create(options.windowsSandboxPath);
-    boolean useWindowsSandbox;
+    boolean windowsSandboxSupported;
     try (SilentCloseable c = Profiler.instance().profile("shouldUseWindowsSandbox")) {
-      useWindowsSandbox = shouldUseWindowsSandbox(options.useWindowsSandbox, windowsSandboxPath);
+      windowsSandboxSupported = shouldUseWindowsSandbox(options.useWindowsSandbox, windowsSandboxPath);
```

Comment:
To my surprise, checking `OS.getCurrent()` doesn't seem to be needed in `shouldUseWindowsSandbox()`. Do you know why?

Reply:
`options.useWindowsSandbox` is by default `no`, so this function will just return `false` without doing anything.